test/hotspot/jtreg/TEST.quick-groups

```diff
@@ -1784,7 +1784,6 @@ vmTestbase_vm_compiler_quick = \
   vmTestbase/jit/t/t111/t111.java \
   vmTestbase/jit/t/t112/t112.java \
   vmTestbase/jit/t/t113/t113.java \
-  vmTestbase/jit/tiered/Test.java \
```

Comment:
Why this test removed?

Reply:
It was an odd test that check if the -XX:+PrintTieredEvents flag didn't work with -XX:-TieredCompilation. Since now it always works that test doesn't make much sense anymore.